### PR TITLE
Adding event registration button logic.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Styles event registration link as button, and wraps hyperlinks in description in <a href>
Task: https://va-gov.atlassian.net/browse/VAGOV-4154

### CODE REVIEW

File: src/site/filters/liquid.js [hunk 1]
```diff
@@ -61,6 +73,41 @@ module.exports = function registerFilters() {
     return `https://img.youtube.com/vi/${string}/sddefault.jpg`;
   };
 
+  liquid.filters.outputLinks = data => {
```
Comment (jbalboni) [inline src/site/filters/liquid.js]: Do you have an example of what the data looks like for this?

Comment (ethanteague) [reply src/site/filters/liquid.js]: You bet, @jbalboni - it's just plain text: ```Email us: metrowebz@gmail.com
www.google.com
http://google.com
941-915-1811```
then after filter treatment: ```Email us: <a target="_blank" href="mailto:metrowebz@gmail.com">metrowebz@gmail.com</a>
<a href="http://www.google.com" target="_blank" rel="noopener ">www.google.com</a>
<a href="http://google.com" target="_blank" rel="noopener ">http://google.com</a>
<a href="tel:941-915-1811">Phone: 941-915-1811</a>```

Comment (ethanteague) [reply src/site/filters/liquid.js]: Updated per convo in slack to just use phone search. Field is now rich text, @jbalboni

File: src/site/stages/build/drupal/graphql/eventPage.graphql.js [hunk 2]
```diff
@@ -60,6 +64,10 @@ module.exports = `
       }
     }
     fieldEventRegistrationrequired
-    fieldAdditionalInformationAbo
+    ${
+      enabledFeatureFlags[featureFlags.FEATURE_FIELD_ADDITONAL_INFO]
+        ? 'fieldAdditionalInformationAbo'
+        : 'fieldAdditionalInformationAbo {processed}'
```
Comment (jbalboni) [inline src/site/stages/build/drupal/graphql/eventPage.graphql.js]: I think this is reversed? So far we've been associating the query change that will be in place going forward with the feature flag being true.

File: src/site/utilities/featureFlags.js [hunk 3]
```diff
@@ -35,6 +39,7 @@ const flagsByBuildtype = {
     featureFlags.FEATURE_REGION_DETAIL_PAGE_FEATURED_CONTENT,
   ],
   vagovstaging: [
+    featureFlags.FEATURE_FIELD_ADDITIONAL_INFO,
```
Comment (jbalboni) [inline src/site/utilities/featureFlags.js]: I think we want to remove this flag from staging and prod, since the change isn't in those environments yet.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**